templates/base.html.twig

```diff
@@ -41,6 +41,15 @@
         {% block extra_link '' %}
 
         <title>{% block title %}{% block page_title '' %} | {% block app_title 'La République En Marche !' %}{% endblock %}</title>
+        {% if app.environment == 'prod' %}
+            <!-- Google Tag Manager -->
+            <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
```

Comment:
Should we add this script at the end of the page to avoid slowing down the page loading ?
Same question for the other one.

Reply:
Nop, as high as possible actually, to let the tracker be loaded before the tracked elements.

Reply:
Alright :+1: